src/components/SavedMarkdownList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+export default function MarkdownList({ markdownList }) {
```

Comment:
If you only are going to display the titles, why no just pass titles instead of the entire file.

Reply:
You would then create a `getTitles` selector

Reply:
> You would then create a `getTitles` selector

We will add a selector to get a list of titles in the container component work.
Thank you,
Lance